rezerwacje-app/build.gradle

```diff
@@ -23,12 +23,14 @@ repositories {
 }
 
 dependencies {
-	compile 'org.postgresql:postgresql'
+	runtimeOnly 'org.postgresql:postgresql'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	compileOnly 'org.projectlombok:lombok'
 	developmentOnly 'org.springframework.boot:spring-boot-devtools'
 	annotationProcessor 'org.projectlombok:lombok'
+
+	testCompile group: 'com.h2database', name: 'h2', version: '1.3.148'
```

Comment:
That's good idea to add versioning in the dependencies for compatibility. What about adding versions to the rest? But I would keep it in one style either "group:name:version" or separate as you did in h2.